Doc/library/socket.rst

```diff
@@ -631,6 +647,18 @@ Constants
 
    .. versionadded:: 3.12
 
+.. _socket-ethernet-types:
+
+.. data:: ETHERTYPE_ARP
+          ETHERTYPE_IP
+          ETHERTYPE_IPV6
+          ETHERTYPE_VLAN
+
+   IEEE 802.3 protocol numbers.
+
```

Comment:
```suggestion

.. availability:: Unix

```

Appears to be missing here?

Reply:
This is available on Linux, FreeBSD and macOS

Reply:
Does that not count as `Unix`, then, for practical purposes? Either way, it should be documented as such.